.circleci/config.yml

```diff
@@ -0,0 +1,20 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: debian
+
+    steps:
+      - checkout
+
+      - run:
+          name: install dependencies
+          command: |
+            apt update && apt install --yes $(cat .circleci/dependencies.list)
```

Comment:
Rather than installing dependencies each time, it would be better to create a [custom Docker image](https://circleci.com/docs/2.0/custom-images/) with the dependencies already installed.

Reply:
Addressed by #2.